lib/ansible/plugins/action/assemble.py

```diff
@@ -51,7 +51,7 @@ def _assemble_from_fragments(self, src_path, delimiter=None, compiled_regexp=Non
             if not os.path.isfile(fragment) or (ignore_hidden and os.path.basename(fragment).startswith('.')):
                 continue
 
-            fragment_content = open(self._loader.get_real_file(fragment), 'rb').read()
+            fragment_content = open(self._loader.get_real_file(fragment, decrypt=decrypt), 'rb').read()
```

Comment:
Where does the RHS decrypt come from?  It appears to be undefined here.

Needs to be passed to _assemble_from_fragments as an kwarg I assume.